spec/factories/early_career_teacher_profiles.rb

```diff
@@ -4,5 +4,9 @@
   factory :early_career_teacher_profile do
     user
     school
+
+    trait :sparsity_uplift do
+      sparsity_uplift { true }
+    end
```

Comment:
Would be good to do this for `pupil_premium_uplift` too.

Reply:
Added